.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,11 @@
-## What kind of a pull request is this?
-
-- QA / Code Review
-<!-- Add other options if appropriate -->
-
+## Description of changes
+<!-- It is often obvious what changed by looking at the code, so it is more helpful to say _why_ it should be changed -->
+<!-- If the Pull Request is not ready to be merged, please use a draft pull request -->
 
 ## Code Checklist
 
-- [ ] The code follows dotkom code style 
-- [ ] The code passes the defined tests
 - [ ] I have added tests for the code I added
```

Comment:
Ref the comment I make below:
In this case most people should make tests so I agree with having a checkbox here. It makes the mental difficulty of merging code with no tests higher.